app/design/adminhtml/default/default/layout/catalog.xml

```diff
@@ -32,7 +32,8 @@
     <adminhtml_catalog_product_new>
         <update handle="editor"/>
         <reference name="head">
-            <action method="addItem"><type>js</type><name>html5sortable.min.js</name><params/></action>
+            <action method="addJs"><name>Sortable.min.js</name></action>
```

Comment:
would it be awful to rename it to lowercase? for some reason case-sensitive names for static assets make me uncomfortable 😅 although it should never be a problem, I don't know, a single uppercase name seems like a sore thumb

Reply:
Fixed. I didn't like the uppercase either.